string key1
string key2
float32 meas_range